### DISCUSSION TITLE
Ranked choice voting - Why options ranked 1st show 0% in the results?

### DISCUSSION THREAD

**Discussion by zhiganov** [snapshot-labs/snapshot#1624] [Q&A] [answered]
Hi guys. Yesterday I set up a voting for our DAO. We decided to let members rank a few options. 
Now I look at the votes and see that some options that were ranked 1st are not counted at all... They have 0% in the results.
Does it have something to do with the voting mechanics?

It looks like snapshot is rebalancing votes in real time, killing off options that don't have a chance...
But then it's getting less and less useful with time... As options that don't win in the first minutes or hours of voting don't have a chance... Even though most of the voters may come a day later, they just wouldn't want to waste their voting power on options that have 0%.
Is it a bad implementation or I'm doing something wrong?

Link to the voting page: https://snapshot.org/#/proteincommunity.eth/proposal/0xa66b633313d23308aa36a7b68ccc711e17eeaa49754cbac08a8c91a2387db4a2

**Comment by samwhosung** [comment, marked as answer]
This is expected behavior, results are calculated via IRV. 

https://docs.snapshot.org/proposals/voting-types#ranked-choice-voting-irv

https://en.wikipedia.org/wiki/Instant-runoff_voting

**Reply by zhiganov** [reply]
Is there any way to export votes? Like in CSV or something?

**Reply by samwhosung** [reply]
We've recently removed it because of scaling concerns. We are planning to work on this feature back though

**Reply by samwhosung** [reply]
Until then you could use this query https://hub.snapshot.org/graphql?operationName=Votes&query=query%20Votes%20%7B%0A%20%20votes%20(%0A%20%20%20%20first%3A%201000%0A%20%20%20%20skip%3A%200%0A%20%20%20%20where%3A%20%7B%0A%20%20%20%20%20%20proposal%3A%20%220x42605fa85d059459f7072aeaeea4cd14ada78e15d9aa1998fe99634a75f9f5a6%22%0A%20%20%20%20%7D%0A%20%20%20%20orderBy%3A%20%22created%22%2C%0A%20%20%20%20orderDirection%3A%20desc%0A%20%20)%20%7B%0A%20%20%20%20id%0A%20%20%20%20voter%0A%20%20%20%20created%0A%20%20%20%20proposal%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%7D%0A%20%20%20%20choice%0A%20%20%20%20space%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A

**Reply by zhiganov** [reply]
thank you very much!

**Comment by zhiganov** [comment]
Thank you for the answer. I read snapshot documentation before posting my question. It's not a good explanation tbh. I guess this mechanic isn't suited for voting that lasts more than a few hours.

**Reply by samwhosung** [reply]
Why? This is a common voting mechanic in elections that go on for days.

**Reply by zhiganov** [reply]
Don't you agree that it makes options that have 0% less likely to get new votes?
Perhaps if the results were hidden for voters, it would make more sense?

**Reply by samwhosung** [reply]
Oh I definitely agree that blind voting should be used with RCV. On this we are also working
